packages/tables/resources/views/index.blade.php

```diff
@@ -618,7 +619,7 @@ class="filament-tables-record-action-button flex-1 block py-3"
                                 @endif
                             @endif
 
-                            @if ($isSelectionEnabled)
+                            @if ($isSelectionEnabled && $selectRecordsPosition === \Filament\Tables\Actions\SelectRecordsPosition::RowStart)
```

Comment:
Could you import the FQN at the top of file just like the usage of actions position?